test/ibm/test_random.py

```diff
@@ -108,6 +108,12 @@ def setUpClass(cls, provider):
         random_service._initialized = False
         cls.provider._random = random_service
 
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """Class level teardown."""
+        super().tearDownClass()
+        cls.provider._disable_account()
```

Comment:
`_disable_account()` is a class method, I'd prefer to access it via `IBMProvider`.